packages/flutter_driver/lib/src/extension.dart

```diff
@@ -73,15 +75,16 @@ class _FlutterDriverExtension {
     });
 
     _commandDeserializers.addAll(<String, CommandDeserializerCallback>{
-      'get_health': (Map<String, dynamic> json) => new GetHealth.deserialize(json),
-      'get_render_tree': (Map<String, dynamic> json) => new GetRenderTree.deserialize(json),
-      'tap': (Map<String, dynamic> json) => new Tap.deserialize(json),
-      'get_text': (Map<String, dynamic> json) => new GetText.deserialize(json),
-      'scroll': (Map<String, dynamic> json) => new Scroll.deserialize(json),
-      'scrollIntoView': (Map<String, dynamic> json) => new ScrollIntoView.deserialize(json),
-      'setInputText': (Map<String, dynamic> json) => new SetInputText.deserialize(json),
-      'submitInputText': (Map<String, dynamic> json) => new SubmitInputText.deserialize(json),
-      'waitFor': (Map<String, dynamic> json) => new WaitFor.deserialize(json),
+      'get_health': (Map<String, String> json) => new GetHealth.deserialize(json),
```

Comment:
this doesn't seem right. A JSON deserialiser is going to return a `Map<String, dynamic>`, even if we "know" it'll "always" be strings. Unless `GetHealth.deserialize` et al actually went and verified each entry is a `String` and then mapped it to a `Map` with the right type.